src/Form/selectors-css.js

```diff
@@ -26,7 +27,7 @@ input[autocomplete=email]`
 const GENERIC_TEXT_FIELD = `
 input:not([type=button]):not([type=checkbox]):not([type=color]):not([type=date]):not([type=datetime-local]):not([type=datetime]):not([type=file]):not([type=hidden]):not([type=month]):not([type=number]):not([type=radio]):not([type=range]):not([type=reset]):not([type=search]):not([type=submit]):not([type=time]):not([type=url]):not([type=week])`
 
-const password = `input[type=password]:not([autocomplete*=cc]):not([autocomplete=one-time-code])`
+const password = `input[type=password]:not([autocomplete*=cc]):not([autocomplete=one-time-code]):not([name*=answer i])`
```

Comment:
Security questions can use a `type=password` field. This fixes one of the known issues in our test cases.